pkg/apis/pingcap.com/v1alpha1/types.go

```diff
@@ -270,25 +286,31 @@ type TiKVFailureStore struct {
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
+// +k8s:openapi-gen=true
 // Backup is a backup of tidb cluster.
 type Backup struct {
-	metav1.TypeMeta   `json:",inline"`
+	metav1.TypeMeta `json:",inline"`
+	// +k8s:openapi-gen=false
 	metav1.ObjectMeta `json:"metadata"`
 
-	Spec   BackupSpec   `json:"spec"`
+	Spec BackupSpec `json:"spec"`
+	// +k8s:openapi-gen=false
 	Status BackupStatus `json:"status"`
 }
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
+// +k8s:openapi-gen=true
 // BackupList contains a list of Backup.
 type BackupList struct {
 	metav1.TypeMeta `json:",inline"`
```

Comment:
Why add "// +k8s:openapi-gen=false" in L83 but not here?

Reply:
ditto